### PR TITLE
Clean up spark docs

### PR DESCRIPTION
**Pull request by shadowhand**
A number of minor fixes:

- remove unnecessary slashes
- use `Acme` namespace universally instead of `My`
- fix middleware value adding references

Refs #95

### CODE REVIEW

**Comment by shadowhand** [comment]
@elazar :8ball:

File: docs/index.md [hunk 1]
```diff
@@ -125,29 +125,29 @@ Here's an example of what this `index.php` file might look like.
...
-    '\\Spark\\Middleware\\DefaultCollection'
+    'Spark\Middleware\Collection',
+    'Spark\Middleware\DefaultCollection'
 );
 
 // Configure the router
 $injector->prepare(
-    '\\Spark\\Router',
-    function(\Spark\Router $router) {
+    'Spark\\Router',
```
Comment (bvisness) [inline docs/index.md]: Looks like you missed the double backslashes here.

File: docs/index.md [hunk 2]
```diff
@@ -240,11 +240,11 @@ $configuration->apply($injector);
 The router maps URIs to the corresponding [domain](https://github.com/pmjones/adr#model-vs-domain) that the action should use. This is implemented in the Spark [`Router`](https://github.com/sparkphp/spark/blob/master/src/Router.php) class. Here is an example of what configuring an instance of it could look like.
 
 ```php
-use MyApp\Domain;
+use Acme\Domain;
 
 $injector->prepare(
-    '\\Spark\\Router',
-    function(\Spark\Router $router) {
+    'Spark\\Router',
```
Comment (bvisness) [inline docs/index.md]: And here.

File: docs/index.md [hunk 3]
```diff
@@ -281,12 +281,12 @@ class Routes
 
     protected function get($uri, $domain)
     {
-        $this->router->get($uri, 'MyApp\\Domain\\' . $domain);
+        $this->router->get($uri, 'Acme\Domain\\' . $domain);
```
Comment (bvisness) [inline docs/index.md]: And here.

Comment (shadowhand) [inline docs/index.md]: This one is required because otherwise it would be escaping the `'` which would blow the whole thing up.

Comment (shadowhand) [inline docs/index.md]: Also, this is why I strongly prefer never to create class names with string interpolation.

Comment (bvisness) [inline docs/index.md]: Oops, gotcha.

File: docs/index.md [hunk 4]
```diff
@@ -125,29 +125,29 @@ Here's an example of what this `index.php` file might look like.
...
 );
 
 // Bootstrap the application
-$dispatcher = $injector->make('\\Relay\\Relay');
+$dispatcher = $injector->make('Relay\Relay');
 $dispatcher(
-    $injector->make('Psr\\Http\\Message\\ServerRequestInterface'),
-    $injector->make('Psr\\Http\\Message\\ResponseInterface')
+    $injector->make('Psr\Http\Message\ServerRequestInterface'),
+    $injector->make('Psr\Http\Message\ResponseInterface')
```
Comment (elazar) [inline docs/index.md]: This is going to [conflict](https://github.com/sparkphp/spark/pull/95/files#diff-1a523bd9fa0dbf998008b37579210e12R127) with #95.

File: docs/index.md [hunk 5]
```diff
@@ -217,19 +221,15 @@ Finally, you must apply the configuration set you've created to the injector in
...
-$configuration = new \My\ConfigurationSet;
+$configuration = new Acme\Configuration\ConfigurationSet;
 
-// If you've created an instance of the default configuration set, use it inline
-$configuration = new \Spark\Configuration\DefaultConfigurationSet([
-    '\\My\\FooConfiguration',
-    // ...
-]);
+// Or if you simply want to extend the default set:
+// $configuration = new Spark\Configuration\DefaultConfigurationSet([ /* ... */ ]);
```
Comment (elazar) [inline docs/index.md]: I think it's better to leave this line uncommented for syntax highlighting when it's displayed. The comment above should be sufficient to indicate the either/or situation.

File: docs/index.md (hunk 4 above)
Comment (shadowhand) [inline docs/index.md]: Yeah that's fine. If this is good, let's merge it and then I'll resolve the conflicts with #95.

File: docs/index.md (hunk 5 above)
Comment (shadowhand) [inline docs/index.md]: :ok:

**Comment by elazar** [comment]
:+1:

**Pull request merged**